src/modules/function.js

```diff
@@ -0,0 +1,73 @@
+import { addTask, updateLocalStorage } from './app.js';
+
+const removeTask = (tasks, taskId) => {
+  if (taskId >= 0 && taskId < tasks.length) {
+    tasks.splice(taskId, 1);
+    updateLocalStorage(tasks);
+  }
+};
+
+const clearCompletedTasks = (tasks) => {
+  const remainingTasks = tasks.filter((task) => !task.completed);
+  tasks.length = 0;
+  remainingTasks.forEach((task) => tasks.push(task));
+  updateLocalStorage(tasks);
+};
+
+export default function populateTodoList() {
```

Comment:
Consider splitting the populateTodoList() function into smaller functions that handle specific tasks, which can make the code more readable and maintainable.